e2e/libs/backupstore/base.py

```diff
@@ -0,0 +1,198 @@
+from abc import ABC, abstractmethod
+import os
+import time
+import hashlib
+
+SETTING_BACKUP_TARGET = "backup-target"
+SETTING_BACKUP_TARGET_CREDENTIAL_SECRET = "backup-target-credential-secret"
+SETTING_BACKUPSTORE_POLL_INTERVAL = "backupstore-poll-interval"
+
+BACKUPSTORE_BV_PREFIX = "/backupstore/volumes/"
+BACKUPSTORE_LOCK_DURATION = 150
+
+RETRY_COUNT = 300
+RETRY_INTERVAL = 1
```

Comment:
I'm still not sure about this. WDYT? @longhorn/qa

Reply:
As the test grows, it often accumulates numerous constants, like [this](https://github.com/longhorn/longhorn-tests/blob/master/manager/integration/tests/common.py#L30-L302). I believe it would be more organized to manage these contents is dedicated 
 constant file.
 
However, if the team decides not to take this approach, I will understand that decision. As such, we then need to take out the constants in the constant.py files and place them in the relevant places.

Reply:
It's because `common.py` is written in C-like style and mixing everything together. We can use a more OOP way to handle them. And I think it is reasonable for constants placed in different files if they are specific to and only used by a particular class.

Reply:
I personally prefer the constants in a separate file for easier management, but I am flexible if the team decides otherwise. I believe what's most important is making sure that @longhorn/qa agrees and understands the final decision and that we maintain a consistent convention in the e2e.

Reply:
Based on my previous experience, my habit is similar to @c3y1huang's suggested usage approach, where I centralize the use of variables in the same variable file, `constants.py`. In the future, when the data in the variable file becomes extensive, it will be easier for us to organize and categorize them further.
However, I currently don't have a concept of the method @yangchiu mentioned: `We can use a more OOP way to handle them.`

Reply:
Refined. Thank you!